shared/actions/platform-specific/push.native.js

```diff
@@ -70,6 +74,18 @@ const listenForNativeAndroidIntentNotifications = emitter => {
     emitter(PushGen.createNotification({notification: lastPushForAndroid}))
     lastPushForAndroid = null
   })
+
+  // TODO: move this out of this file.
+  RNEmitter.addListener('onShareData', (evt) => {
+    logger.info('[ShareDataIntent]', evt)
+    emitter(RouteTreeGen.createNavigateTo({path: FsShared.fsRootRoute}))
+    emitter(FsGen.createSetIncomingShareLocalPath({localPath: FsTypes.stringToLocalPath(evt.path)}))
+    emitter(FsGen.createShowIncomingShare({initialDestinationParentPath: FsTypes.stringToPath('/keybase')}))
+  })
+  RNEmitter.addListener('onShareText', (evt) => {
+    logger.info('[ShareTextIntent]', evt)
+    emitter(RouteTreeGen.createNavigateTo({path: FsShared.fsRootRoute}))
```

Comment:
If we're not handling text, can we disable it on java side?

Reply:
We will handle text, this is mostly a placeholder. I'll add a comment to that effect.